src/Libraries/Microsoft.Extensions.Diagnostics.Extra/Logging/LoggingRedactionExtensions.cs

```diff
@@ -19,12 +21,38 @@ public static class LoggingRedactionExtensions
     /// <param name="builder">The dependency injection container to add logging to.</param>
     /// <returns>The value of <paramref name="builder"/>.</returns>
     public static ILoggingBuilder EnableRedaction(this ILoggingBuilder builder)
+        => EnableRedaction(builder, _ => { });
+
+    /// <summary>
+    /// Enables redaction functionality within the logging infrastructure.
+    /// </summary>
+    /// <param name="builder">The dependency injection container to add logging to.</param>
+    /// <param name="configure">Delegate the fine-tune the options.</param>
+    /// <returns>The value of <paramref name="builder"/>.</returns>
+    public static ILoggingBuilder EnableRedaction(this ILoggingBuilder builder, Action<LoggerRedactionOptions> configure)
     {
         _ = Throw.IfNull(builder);
 
         builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<ILoggerFactory, ExtendedLoggerFactory>());
+        _ = builder.Services.Configure(configure);
         _ = builder.Services.AddOptions<LoggerRedactionOptions>();
```

Comment:
Can be removed, `Configure()` calls `AddOptions()`

Reply:
Done.